### ISSUE TITLE
Black viewfinder in newer versions LG G7

### ISSUE THREAD

**Issue by hshsa** [mirsadm/motioncam#170]
Hello, in the latest stable version that I download on December 30th it worked perfectly fine but I needed the feature that keeps the last opened mode 

So I downloaded all betas from beta 0 8.0.7 and in all versions the viewfinder is black. Captured photos are normal however

**Comment by mirsadm** [comment]
Do you use dual exposure? In photo mode you can see the viewfinder but not in video mode?

**Comment by hshsa** [comment]
No, I don't use dual exposure. In 8.0.4 viewfinder is NOT black but if dual exposure is on than the viewfinder is black

In 8.0.7 with default settings the viewfinder is black at all mods (night, burst, video, photo). I tried to switch on dual exposure in this version hoping that it will fix the black viewfinder but it didn't.

**Comment by mirsadm** [comment]
Is this still a problem? If so I'll reopen the issue

**Comment by hshsa** [comment]
Yes, it's an issue, reopen please

I want to use your new versions because of the "last opened mode" feature but I can't because of black viewfinder

**Comment by mirsadm** [comment]
does switching cameras help?

**Comment by hshsa** [comment]
I switched to front and then back - still black viewfinder even on the front camera. There's no buttons to switch to ultra wide camera if you ask about that

**Comment by mirsadm** [comment]
I am hoping this is fixed in the latest beta. Let me know how you go.

**Comment by hshsa** [comment]
> I am hoping this is fixed in the latest beta. Let me know how you go.

Hello, nope. I also noticed there's no more DE (dual exposure) toggle in settings. In 8.0.4 (the version with normal viewfinder) it caused black screen if turned on, in 8.0.7 viewfinder was black by default although the DE was off and in the newest version the viewfinder is still black + DE is permanent, can't even turn it off/on

The app takes photographs though, no problem with that
